packages/eslint-plugin-taro/rules/no-spread-in-props.js

```
@@ -0,0 +1,29 @@
+module.exports = {
+  meta: {
+    docs: buildDocsMeta(ERROR_MESSAGE, 'no-spread-in-props')
+  },
+
+  create (context) {
+    return {
+      JSXSpreadAttribute (node) {
+        const parents = context.getAncestors(node)
+        const jsx = parents.find(p => p.type === 'JSXOpeningElement')
+        if (jsx && jsx.name && jsx.name.name) {
+          const componentName = jsx.name.name
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)